Warning to include "http://" when accessing the server (#132)
learningequality/kolibri-docs: docs/access.rst

  * You can also use the ``ipconfig`` command on Windows or ``ifconfig`` command on Linux/OSX to find the externally visible IP address of the device running the Kolibri.

.. warning::

>   * When limited to using older browsers it may be necessary to **explicitly include** the ``http://`` in front of the IP in the address bar, for example ``http://192.168.8.134:8080``. Many modern browsers may also take the IP address or the local network hostname as a search term and redirect you to Google. In other cases, browsers may try to use the secure **HTTPS** protocol by default and add an **S**  automatically (``httpS://``), which may also cause issues while running Kolibri.

Reviewer: I don't thin `http://192.168.0.134:8080` is a good example. The browser will redirect to https when the port is not used. I.E: `http://192.168.0.134`will be redirect to `https://192.168.0.134` because the port is not added. If the port is added it keeps `http`.
That's why I mentioned the problem exists *only* when using port *80*, because this was the former implicit port and now it has been changed to *443* (https) by the browsers since they implemented https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security

Reviewer 2: I'm not sure how browsers will implement this in the future, but I think the warning is a reasonable caution such that everyone can go ahead and make their own observations / troubleshooting with some pretty solid hints.

Reviewer 3: We can address Jose's feedback by adapting the text slightly

```suggestion
  * When accessing Kolibri by IP, in many browsers it is necessary to **explicitly include** the ``http://`` in front of the IP in the address bar. For example, this might be ``http://192.168.8.134:8080`` for port ``8080`` or ``http://192.168.8.134`` for port ``80``. Omitting ``http://`` may result in undesirable redirects.
```

Reviewer 2: I already made a suggestion for this, see below ;)

>   * When limited to using older browsers it may be necessary to **explicitly include** the ``http://`` in front of the IP in the address bar, for example ``http://192.168.8.134:8080``. Many modern browsers may also take the IP address or the local network hostname as a search term and redirect you to Google. In other cases, browsers may try to use the secure **HTTPS** protocol by default and add an **S**  automatically (``httpS://``), which may also cause issues while running Kolibri.

Reviewer 2: ```suggestion
  * When typing in your server's address in the browser address bar, it may be required to add `http://` in the first part of the address, for instance `http://1.2.3.4:8080` or `http://1.2.3.4:80`. Many modern browsers may understand an IP address or local network hostname as a search term and redirect a user to Google. In other cases, browsers may add `httpS://` automatically, which will also cause issues.
```

@radinamatic notice that the original issue pertained some kind of "older browsers" and @jredrejo is referring to current and upcoming browsers. I think it's best to just describe the behavior and add a simple-to-read caution.

If people are setting up Kolibri on Port 80 and configuring their own host names, i.e. `http://kolibri`, they will probably understand the caution, given that they have a bit of technical background required to make a choice of port 80 :)
